chain/jsonrpc/src/lib.rs

```diff
@@ -286,6 +287,26 @@ impl JsonRpcHandler {
         Ok(Value::String("".to_string()))
     }
 
+    #[cfg(feature = "adversarial")]
```

Comment:
I wonder if we want to move all the adversarial methods into a second `impl JsonRpcHandler` (you can have more than one `impl` blocks for a type), so we feature-gate the whole `impl` instead of individual methods :thinking:

Reply:
It makes sense to me.

Reply:
Not related to this PR directly, but I guess it's okay to do it here.